config/heroes/special.json

```diff
@@ -25,11 +29,9 @@
 		"skills":
 		[
 			{ "skill" : "wisdom", "level": "basic" },
-			{ "skill" : "fireMagic", "level": "expert" } ],
-		"specialties":
-		[
-			{ "type":11, "val": 14, "subtype": 0, "info": 3 }
-		]
+			{ "skill" : "fireMagic", "level": "expert" }
+		],
+		"specialty" : { "bonuses" : {  } }
```

Comment:
Something missing here

Reply:
No, that's correct. Adrienne's "specialty" is that she starts with expert fire magic.
http://heroes.thelazy.net/wiki/Adrienne

Reply:
OK, then comment is missing :)

Reply:
I agree, but I don't think the Json parser permits comments?

Reply:
`//`-style comments allowed

Reply:
Right, just noticed there are comments like this right at the beginning of the file. Will add.